web/layouts/_default/rules-table.html

```
@@ -0,0 +1,127 @@
+                            </div>
+                        </div>
+                        <div class="flex flex-none items-center gap-x-4">
+                            {{- if $rule.Enabled }}
+                            <p class="mt-0.5 rounded-md bg-green-50 px-1.5 py-0.5 text-xs font-medium text-green-700 ring-1 ring-inset ring-green-600/20">Enabled</p>
+                            {{- else }}
+                            <p class="mt-0.5 rounded-md bg-gray-50 px-1.5 py-0.5 text-xs font-medium text-gray-600 ring-1 ring-inset ring-gray-500/10">Disabled</p>
+                            {{- end }}
+
+                            {{ if $rule.CanEdit }}
+                            {{ if gt (len $.Params.Rules) 1 }}
+                            <button type="button"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)